.github/CONTRIBUTING.md

```diff
@@ -93,11 +93,10 @@ cmake --build build -j4
 
 Tips:
 
-* You can use `virtualenv` (from PyPI) instead of `venv` (which is Python 3
-  only).
+* You can use `virtualenv` (faster, from PyPI) instead of `venv`
 * You can select any name for your environment folder; if it contains "env" it
   will be ignored by git.
-* If you don’t have CMake 3.14+, just add “cmake” to the pip install command.
+* If you don’t have CMake 3.14+, just add "cmake" to the pip install command.
```

Comment:
@henryiii I know some commit hooks we can add to ban smart quotes, if you want.

Reply:
:heart: (though a simple pygrep would do it, as well, I'd think)

Reply:
@henryiii It also banned some smart apostophe, we can tweak the commit hook to allow them if you want.